Makefile

```diff
@@ -176,14 +178,27 @@ else
 KUSTOMIZE=$(shell which kustomize)
 endif
 
+OPERATOR_SDK = $(BINDIR)/operator-sdk
```

Comment:
not working for me 
```
operator-sdk:
	@@echo $(CURRDIR)
```

```
# make operator-sdk && echo 1

1
```

Reply:
you might need to set the CURRDIR value.
CURRDIR = $(shell pwd)

Reply:
- well, that's what I did `BINDIR=$(shell pwd)/bin` in the first place. 
- I believe Ohad might've gave the comment as `CURRDIR` will be available by default

Reply:
`pwd` is problematic because it will point to the directory where the make was invoked not where the makefile resided.
`CURDIR` is supposed to be a built-in and will always point to the directory where the make file resides.

I think the issue is that I misspelled CURDIR as CURRDIR in my last comment